src/mmw/mmw/settings/layer_settings.py

```diff
@@ -233,6 +233,33 @@
         'opacity': 0.618,
         'has_opacity_slider': True,
         'perimeter': pa_perimeter,
+    },
+    {
+        'code': 'drb_catchment_water_quality_tn',
+        'display': 'DRB Catchment Water Quality Data TN Loading Rates',
+        'table_name': 'drb_catchment_water_quality_tn',
+        'vector': True,
+        'overlay': True,
+        'minZoom': 3,
+        'perimeter': drb_perimeter
+    },
+    {
+        'code': 'drb_catchment_water_quality_tp',
+        'display': 'DRB Catchment Water Quality Data TP Loading Rates',
+        'table_name': 'drb_catchment_water_quality_tp',
+        'vector': True,
+        'overlay': True,
+        'minZoom': 3,
+        'perimeter': drb_perimeter
+    },
+    {
+        'code': 'drb_catchment_water_quality_tss',
+        'display': 'DRB Catchment Water Quality Data TSS Loading Rates',
+        'table_name': 'drb_catchment_water_quality_tss',
+        'vector': True,
```

Comment:
That makes sense. Another consideration here would be which other layers we might want to allow/prevent from being turned on when these layers are on: would it be preferable to allow these layers to be on in tandem with the boundary layers or in tandem with the coverage layers.